client/src/components/registration/components/App.js

```diff
@@ -0,0 +1,41 @@
+import React, { useState, useEffect } from "react";
+import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
+import { hot } from "react-hot-loader/root";
+
+import getCurrentUser from "../services/getCurrentUser";
+import "../assets/scss/main.scss";
+import RegistrationForm from "./registration/RegistrationForm";
+import SignInForm from "./authentication/SignInForm";
+import TopBar from "./layout/TopBar";
+
+const App = (props) => {
```

Comment:
this looks like you have made a new `App` component in the `registration` folder? remember to review all the changes you've made before submitting a PR!

I'm very confused, because it looks like a lot of files/ folders were duplicated inside this `registration` folder?